commands/droplet_actions.go

```diff
@@ -268,6 +292,63 @@ func RunDropletActionDisableBackups(c *CmdConfig) error {
 	return performAction(c, fn)
 }
 
+func readDropletBackupPolicy(c *CmdConfig) (*godo.DropletBackupPolicyRequest, error) {
```

Comment:
This function used in `droples.go` and in `droplet_actions.go`, but I keep it in `droplet_actions.go`.
Please, let me know if there is a better place to store it.